src/components/variant-aspects-layout/variant-aspects-layout.utils.ts

```diff
@@ -0,0 +1,189 @@
+import { Ref, RefObject, useImperativeHandle, useRef } from 'react'
+import { Layout } from 'react-grid-layout'
+
+import { TAspectDescriptor } from '@service-providers/dataset-level/dataset-level.interface'
+import {
+  TVariantAspectsGridHandles,
+  TVariantAspectsGridLayout,
+  TVariantAspectsLayoutGridChangeHandler,
+} from './variant-aspects-layout.interface'
+
+export const GRID_LAYOUT_COLS = 3
```

Comment:
Maybe it is better to move this constants to special file, like `variant-aspects-layout.constants.ts`

Reply:
Agree, better to move to constants.ts

Reply:
moved